### PR TITLE
Add a new page My Returns to the Company Account Dashboard block

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds a new page My Returns to the Company Account Dashboard block.

## Magento release version

- [x] 2.4.x

whatsnew
Added the [My Returns](https://docs.magento.com/user-guide/customers/account-dashboard-my-returns.html) topic, which includes information about managing returns for the customer account on the storefront.

### CODE REVIEW

File: src/customers/account-dashboard-my-returns.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: My Returns
+---
+
+All customer return information is displayed on the **My Returns** page in the customer account.
+
+![]({% link images/images/my-returns-page.png %}){: .zoom}
+_My Returns Page_
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:8]:
```suggestion
_My Returns_
```

File: src/customers/account-dashboard-my-returns.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Create Return Request from customer account
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:10]: Return request is not a proper noun

```suggestion
## Create return request from customer account
```

File: src/customers/account-dashboard-my-returns.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+Only shipped and invoiced orders, with `Complete` status, can be returned.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:12]:
```suggestion
{:.bs-callout-info}
Only shipped and invoiced orders, with `Complete` status, can be returned.
```

File: src/customers/account-dashboard-my-returns.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+1. From the storefront, the customer logs in and opens their customer account page.
+
+1. The customer opens the **My Orders** page and finds the required order with the status `Complete`.
+
+1. Click <span class="btn">View Order</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:18]:
```suggestion
1. Clicks <span class="btn">View Order</span>.
```

File: src/customers/account-dashboard-my-returns.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+1. To create a return request, click <span class="btn">Return</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:20]:
```suggestion
1. To create a return request, clicks <span class="btn">Return</span>.
```

File: src/customers/account-dashboard-my-returns.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+   ![]({% link images/images/my-orders-returns.png %}){: .zoom}
+   _My Orders Page_
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:23]:
```suggestion
   _My Orders_
```

File: src/customers/account-dashboard-my-returns.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Fill in the **Contact Email Address** field.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:25]:
```suggestion
1. Enters the **Contact Email Address** information.
```

File: src/customers/account-dashboard-my-returns.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+1. Sets **Reason to Return** to one of the following:
+
+    - Wrong Color
+    - Wrong Size
+    - Out of Service
+    - Other
+
+    In case of the reason for the return is different from those listed in the **Reason to Return** field, then you can enter your own in the **Other** field.
+
+1. If you need to return more than one item, click <span class="btn">Add item to return</span> and repeat the previous steps.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:50]:
```suggestion
1. If returning more than one item, clicks <span class="btn">Add item to return</span> and repeats the previous steps.
```

File: src/customers/account-dashboard-my-returns.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+    ![]({% link images/images/create-new-return.png %}){: .zoom}
+   _Create New Return Page_
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:53]:
```suggestion
   _Create New Return_
```

File: src/customers/account-dashboard-my-returns.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Also, you can leave an additional comment for the store admin in the **Comment** field.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:55]:
```suggestion
1. (Optional) Enters an additional comment for the store administrator in the **Comment** field.
```

File: src/customers/account-dashboard-my-returns.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+1. To create a return, click <span class="btn">Submit</span>.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-returns.md:57]:
```suggestion
1. Clicks <span class="btn">Submit</span> to create the return request.
```

Comment (ZhovnirMaria) [reply src/customers/account-dashboard-my-returns.md]: @leslietilling, hi
thank you very much for all your help!
Done.

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1185`.

**Pull request merged**